recipes-ni/ni-grpc-device/ni-grpc-device_git.bb

```diff
@@ -10,37 +10,37 @@ LIC_FILES_CHKSUM = "\
 
 
 DEPENDS += "\
-	grpc-native \
 	googletest-native \
+	grpc \
+	grpc-native \
 	nlohmann-json-native \
+	protobuf \
 	protobuf-native \
 	python3-grpcio-tools-native \
 	python3-mako-native \
 	python3-native \
-	utf8cpp-native \
 	python3-schema-native \
+	utf8cpp-native \
 "
 
-PV = "2.4.0"
+PV = "2.6.0"
 
 
 SRC_URI = "\
 	git://github.com/ni/grpc-device.git;name=grpc-device;branch=main;protocol=https \
 	file://ptest \
-	file://0001-Subject-PATCH-CMakeLists.txt-remove-local-protobuf-i.patch \
-	file://0002-CMakeLists-fixup-utf8cpp-library-link.patch \
-	file://0003-CMakeLists-use-native-python3-binaries.patch \
-	file://0001-CMakeLists-find-gpr-library.patch \
+	file://0001-CMakeLists_txt.patch \
```

Comment:
Yeah, I had split up the patch files to not include .github changes and that's the reason the patch files were renamed based on file names instead of commit title.
I have now made all the suggested changes. Marking this comment as resolved.